react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx

```diff
@@ -39,8 +39,12 @@ export function calculateTableHeight(
 }
 
 export function getScrollbarWidth(): number {
-  if (!window || !document || !document.documentElement)
+  const isSSR =
+    typeof document === 'undefined' || typeof document === 'undefined'
```

Comment:
Why are you using the same terms in the logic expression?

Reply:
True :woman_facepalming: I'll launch again fixing this